reflex-dom-core/src/Reflex/Dom/Builder/Immediate.hs

```diff
@@ -834,11 +925,15 @@ hoistTraverseWithKeyWithAdjust base mapPatch updateChildUnreadiness applyDomUpda
     else do
       modifyIORef' parentUnreadyChildren succ
       writeIORef pendingChange (initialUnready, mempty) -- The patch is always empty because it got applied implicitly when we ran the children the first time
-  let result0 = DMap.map (\(Compose (_, _, _, v)) -> v) children0
-      placeholders0 = weakenDMapWith (\(Compose (_, ph, _, _)) -> ph) children0
-      result' = ffor children' $ mapPatch $ \(Compose (_, _, _, r)) -> r
-  liftIO $ writeIORef placeholders $! placeholders0
-  _ <- DMap.traverseWithKey (\_ (Compose (df, _, _, _)) -> Constant () <$ append df) children0
+  let result0 = DMap.map (_child_result . getCompose) children0
+      childInstallations0 = weakenDMapWith (_child_installation . getCompose) children0
```

Comment:
Do you think there may be a strictness issue here?  Previously, we were casing on the (,,,), but now we're using a record selector.  If my understanding is correct, this could potentially lead to the creation of a thunk that would still include the _child_result.